frontend/src/pages/Landing/index.js

```diff
@@ -325,7 +325,7 @@ function Landing() {
       </Helmet>
       <UserContext.Consumer>
         {({ user }) => (
-          <>
+          <div className="margin-left-3">
```

Comment:
Think we want this to stay a fragment without the `margin` class (probably got added in during a merge!)